pkgs/development/tools/yarn2nix-moretea/yarn2nix/default.nix

```diff
@@ -69,7 +69,7 @@ in rec {
     yarnLock,
     yarnNix ? mkYarnNix { inherit yarnLock; },
     offlineCache ? importOfflineCache yarnNix,
-    yarnFlags ? defaultYarnFlags,
+    yarnFlags ? [ ],
```

Comment:
This is not much used in nixpkgs and I am pretty confident that this is a none breaking change. I am still going to remove the extra flags accumulated in other packages after this is merged.

Reply:
We use this in our internal project to remove the `--ignore-scripts` flag when we need to build native extensions for a module. As it is now, it doesn't seem possible to unset any of the default flags so our use case is broken (what's more it's been backported to nixos-22.05).

Reply:
Not sure how we build those in nixpkgs. How about adding a boolean variable wether to run scripts?

Reply:
#198950